packages/datadog-code-origin/index.js

```diff
@@ -36,20 +36,25 @@ function exitTags (topOfStackFunc) {
  * @returns {Record<string, string>}
  */
 function tag (type, topOfStackFunc, limit) {
-  const frames = getUserLandFrames(topOfStackFunc, limit)
+  // The `Error.prepareStackTrace` API doesn't support resolving source maps.
+  // Fall back to manually parsing the stack trace.
+  const dummy = {}
```

Comment:
nit: I'm not sure if `dummy` really tells me what this variable is being used for 🤔

Reply:
It's not being used for anything really. It's just a sacrificial object on which the `stack` property can be added. Instead of writing `new Error('something').stack`, we use the `Error.captureStackTrace()` API to generate a stack property, which has a few advantages.

I don't think it's right to call it `error` either, as it's not an instance of `Error`. Calling it `obj` is not helpful either. According to the `Error.captureStackTrace()`, its first argument is `object` which is described as:

> The object on which to add the `stack` property.

If you have a good idea for a name let me know, and I'll update it.